app/controllers/app.py

```diff
@@ -643,14 +644,55 @@ def get(self, project_id):
         if not is_owner_or_admin(project, current_user_id, current_user_roles):
             return dict(status='fail', message='Unauthorised'), 403
 
-        apps = App.find_all(project_id=project_id)
+        cluster = Cluster.get_by_id(project.cluster_id)
+
+        if not cluster:
+                return dict(status='fail', message=f'cluster with id {project.cluster_id} does not exist'), 404
+
+        kube_host = cluster.host
+        kube_token = cluster.token
+        kube, extension_api, appsv1_api, api_client, batchv1_api, storageV1Api = create_kube_clients(kube_host, kube_token)
+
+        apps = App.find_all(project_id=project_id) 
 
         apps_data, errors = app_schema.dumps(apps)
 
         if errors:
             return dict(status='fail', message=errors), 500
 
-        return dict(status='success', data=dict(apps=json.loads(apps_data))), 200
+        apps_data_list, err = app_schema.loads(apps_data)
+
+        if err:
+            return dict(status='fail', message=err), 500
+        
+        for app in apps_data_list:
+            app_status_object = appsv1_api.read_namespaced_deployment_status(app['alias']+"-deployment",project.alias)
+            app_deployment_status_conditions = app_status_object.status.conditions
+            app_has_db = True
+
+            for deplyoment_status_condition in app_deployment_status_conditions:
+                if deplyoment_status_condition.type == "Available":
+                    app_deployment_status = deplyoment_status_condition.status
+                    app['app_running_status'] = app_deployment_status
+
+
+            try: 
+                app_db_status_object = appsv1_api.read_namespaced_deployment_status(app['alias']+"-postgres-db",project.alias)
+                app_db_state_conditions = app_db_status_object.status.conditions
+
+                for app_db_condition in app_db_state_conditions:
+                    if (app_db_condition.type == "Available"):
+                        app_db_status = app_db_condition.status
+
+                        
+                
+            except client.rest.ApiException as e:
+                app_has_db = False
+            
+            if app_deployment_status == True and app_db_status == True and app_has_db == True:
```

Comment:
Please make this statement pythonic, `app_deployment_status == True`  is not pythonic.

Reply:
Hehe.. my bad... lemmi address this